lib/media-capture-demo.ts

```diff
@@ -53,19 +64,19 @@ export class MediaCaptureDemo extends cdk.Stack {
       
       const sdkBucket = s3.Bucket.fromBucketName(this,'amazon-chime-blog-assets','amazon-chime-blog-assets')
 
-      const sdkLayer = new lambda.LayerVersion(this, 'aws-sdk', {
-        code: new lambda.S3Code(sdkBucket, 'aws-sdk2_924_0.zip'),
-        compatibleRuntimes: [lambda.Runtime.NODEJS_14_X],
-        license: 'Apache-2.0',
-        description: 'aws-sdk Layer',
-      });
+      // const sdkLayer = new lambda.LayerVersion(this, 'aws-sdk', {
+      //   code: new lambda.S3Code(sdkBucket, 'aws-sdk2_924_0.zip'),
+      //   compatibleRuntimes: [lambda.Runtime.NODEJS_14_X],
+      //   license: 'Apache-2.0',
+      //   description: 'aws-sdk Layer',
+      // });
```

Comment:
I would just remove it instead of commenting it out.  If you feel it is important to keep, can you add some comments to the code about why it is there and commented out?